netbox/netbox/configuration.example.py

```diff
@@ -50,6 +60,18 @@
 # BASE_PATH = 'netbox/'
 BASE_PATH = ''
 
+# Cache timeout in seconds. Set to `None` to enforce an infinate timeout. Set to 0 to dissable caching by immediatly
```

Comment:
Do we want to support an infinite timeout? Not sure what the implications would be.

Reply:
As discussed in slack, this is just what the Django Caching API supports out of the box. We will leave it be for now.